apps/generator/package.json

```diff
@@ -25,7 +25,8 @@
     "lint:tpl:validator": "eslint --fix --config ../../.eslintrc ../../.github/templates-list-validator",
     "generate:readme:toc": "markdown-toc -i README.md",
     "generate:assets": "npm run docs && npm run generate:readme:toc",
-    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "postinstall": "node ./scripts/postinstall.js"
```

Comment:
I know we talked about postinstallation as best option, but to be honest, I'm still in doubts. This notice will show up every-time someone installs generator, even if it is a dependency to AsyncAPI CLI - so during AsyncAPI CLI installation - at least I think so - something you would have to verify

this means we will spam all generator users, even the ones not using `ag` 🤔 maybe in the end it is better to have a warning invoked only as part of cli.js code? even if it runs every time someone invokes it? 🤔

Reply:
hmmmm my concern with having a warning only for invoking cli.js is that it would not be obvious? Since a lot of users probably already have cli.js as part of their code's workflow and don't really look at it anymore? But I can see this is also a problem for having a message during installation. 

If spamming users is a problem, I can shorten the message a bit so it would be as annoying. What do you think?